pkg/network/cache/multi.go

```diff
@@ -374,13 +357,23 @@ func (x *multiClient) RawForAddress(addr network.Address, f func(*grpc.ClientCon
 		return err
 	}
 
-	err = c.ExecRaw(f)
+	err = f(c.Conn())
 	if err != nil {
 		x.ReportError(err)
 	}
 	return err
 }
 
+func (x *multiClient) Conn() *grpc.ClientConn {
+	var cc *grpc.ClientConn
+
+	_ = x.iterateClients(context.TODO(), func(c clientcore.Client) error {
```

Comment:
will context be meaningful in the future?

Reply:
I don't think so. We need to get some connection, we're not making any real requests here.

Reply:
ok, just do not like `context.TODO()` in any "release" code